kodilla-stream/src/main/java/com/kodilla/stream/homework/ForumStats.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.stream.homework;
+
+import com.kodilla.stream.User;
+import com.kodilla.stream.UsersRepository;
+
+public class ForumStats {
+    public static void main(String[] args) {
+        double averageNumberOfPostsWhenUserAgeIsUp40 = UsersRepository.getUsersList()
+            .stream()
+            .filter(a -> a.getAge() >= 40 && a.getNumberOfPosts() !=0)
```

Comment:
po co ten drugi warunek? Chyba zero postow wrzucamy do sredniej

Reply:
jak to? Potrzebuję wziąć wiek większy lub równy 40  i ilość postów z kolekcji, a bez ostatniego warunku mnie nie puszcza dalej. Jak jest równy w sensie == to wtedy wychodzi zero

Reply:
nie rozumiem. Masz wziac uzytkowanikow starszych lub rownych 40 lat i policzyc srednia ilosc postow. Zalozmy, ze mamy uzytkownika jednego w wieku 40 lat i ma on zero postow. Dla mnie srednia takich uzytkownikow to zero.